lms/djangoapps/instructor_task/tests/test_tasks_helper.py

```diff
@@ -140,6 +145,70 @@ def test_unicode_user_partitions(self):
         _groups = [group.name for group in self.course.user_partitions[0].groups]
         self.assertEqual(_groups, user_groups)
 
+    def test_content_group_partition(self):
+        """
+        Test that cohort-schemed user partitions are ignored in the
```

Comment:
Nit: the name of the method is "content_group_partition", but the doc string refers to cohort-schemed user partitions. Make the method name consistent (test_cohort_scheme_partition).